### DISCUSSION TITLE
How about the File System

### DISCUSSION THREAD

**Discussion by Yo-yo-ooo** [marceldobehere/MaslOS#13] [General]
I think you should increase your support for file systems

**Comment by marceldobehere** [comment]
Will do once I finish other stuff. There is some stuff which I would like to have first, like a simple audio and networo driver. And I also want to fix the SATA read/write being broken on real hardware too. Other filesystems are not too important for me rn. Especially considering that theres no text editor yet lol. (Which I am working on)
